### PR TITLE
Sign in(FORM)

### PR DESCRIPTION
**Pull request by canilo1**
Hey everyone, I thought it was good practice to create a separate PR for any new features. Technically, the sign-in form is a new feature, even though it's very similar to the sign-up form, so I made a new PR. I hope that doesn't bother anyone.

### CODE REVIEW

**Comment by canilo1** [comment]
image_2024-08-16_181125272

**Comment by ElijahMSmith** [comment]
I'm going to ignore the first three commits for the signup form since they are addressed in the other PR. For something like this that depends on other code that is under review, I would either just stick them together or wait for that to get merged before opening another PR so that those commits don't appear in this PR too. That said, the way you've approached this is fine and it's just a limitation of GitHub so don't worry about it here.

File: frontend/src/pages/frontPage/Panel.jsx
Comment (ElijahMSmith) [inline frontend/src/pages/frontPage/Panel.jsx:11]: Let's not keep any dead code we aren't using.

File: frontend/src/pages/frontPage/Panel.jsx
Comment (ElijahMSmith) [inline frontend/src/pages/frontPage/Panel.jsx]: The other PR I made a request to only show the popup when I click "register", which we should also do here when I click "login". Let's fix it in that branch, merge those changes, then rebase this branch and do the same thing.

File: frontend/src/components/layout/signIn.jsx
Comment (ElijahMSmith) [inline frontend/src/components/layout/signIn.jsx:1]: This should be `SignIn.jsx` for consistency.

File: frontend/src/components/layout/SignIn.module.css
Comment (ElijahMSmith) [inline frontend/src/components/layout/SignIn.module.css:1]: Let's also be consistent here in class names and make them all camelCase.

**Review by ElijahMSmith — CHANGES_REQUESTED** [review]
Let's hold off on this MR until the SignUp form is finished and merged.

**Comment by canilo1** [comment]
The Sign-Up page has everything that the Sign-In format has, so I'm going to close this branch since it's no longer needed. It's merged, by the way.

**Pull request closed without merging**
